### PR TITLE
test(FTL-17138): add unit tests for affinidi-did-resolver-methods

### PR DESCRIPTION
**Pull request by YoussefAWasfy**

### CODE REVIEW

**Review by stormer78 — APPROVED** [review]
LGTM

**Review by robert-affinidi — APPROVED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -28,7 +28,7 @@ regex = "1"
 serde = { version = "1.0", features = ["derive", "rc"] }
 serde_json = "1.0"
 blake2 = "0.10"
-ssi = "0.9"
+ssi = { version = "0.9", features = ["secp384r1"] }
```
Comment (anton-iskryzhytskyi) [inline Cargo.toml:31]: Could you please elaborate on why we need this feature for tests?

**Review by anton-iskryzhytskyi — APPROVED** [review]
LGTM

File: Cargo.toml (hunk 1 above)
Comment (YoussefAWasfy) [reply Cargo.toml:31]: To be able to trigger the unsupported key error.
https://github.com/affinidi/affinidi-did-resolver/pull/33/files#diff-1527caf394ce904addf6f1ed3798fcc5b4c2f65031e4781963b1431b06693856R865

**Pull request merged**